app/tests/integration/nimbus/test_desktop_client_integrations.py

```diff
@@ -148,10 +153,14 @@ def test_check_telemetry_enrollment_unenrollment(
             assert False, "Experiment enrollment was never seen in ping Data"
 
     # check experiment exists, this means it is enrolled
+    assert check_ping_for_experiment(experiment_slug)
```

Comment:
Oh also I think we should also try to keep up the habit of adding that second paramter to the assert that adds a helpful little message about what it means if the assertion fails, like just above it.

Reply:
oh you are missing the extra helpful message for the assert, also this check_ping_for_experiment will always return True, I think I am not getting the purpose of this 🤔